modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/trait/DecodedMessage.java

```diff
@@ -2,11 +2,59 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import de.wuespace.telestion.api.message.JsonMessage;
+import io.vertx.core.Future;
 import io.vertx.core.eventbus.Message;
 import io.vertx.core.json.JsonObject;
 
+/**
+ * A wrapper for a decoded message from the {@link Message Vert.x message} to the {@link JsonMessage} type
+ * and basis {@link Message Vert.x message}
+ *
+ * @param message the basic {@link Message Vert.x message}
+ * @param body    the decoded body of the {@link Message Vert.x message} to the {@link JsonMessage} type
+ * @param <V>     the type of the {@link JsonMessage} to map to
+ * @param <T>     the type of body of the {@link Message Vert.x message}
+ * @see JsonMessage#on(Class, Message)
+ *
+ * @author Ludwig Richter (@fussel178)
+ */
 public record DecodedMessage<V extends JsonMessage, T extends JsonObject>(
 		@JsonProperty V body,
 		@JsonProperty Message<T> message
 ) implements JsonMessage {
+
+	/**
+	 * Composes the given future, which eventually returns the received message, with the returned future
+	 * to map from the {@link Message Vert.x message} to the {@link JsonMessage} type.
```

Comment:
```suggestion
	 * Composes the future to return the received message mapped to the {@link JsonMessage} type.
```

Reply:
Done in [`2cd29d6` (#484)](https://github.com/wuespace/telestion-core/pull/484/commits/2cd29d6f1426056b991dcbdcda1b3d2c3274fe9e)